integration-tests/checkstyle-init.patch

```diff
@@ -1,7 +1,7 @@
 --- a/pom.xml
 +++ b/pom.xml
-@@ -366,6 +366,12 @@
-       <version>1.4.1</version>
+@@ -368,6 +368,12 @@
+       <version>1.4.4</version>
```

Comment:
Hmm, this one is unexpected, if you don't recognize this one, I'll try to remove it @Stephan202 😄

Reply:
This is a change in the patch context, not a change in the changes made by the patch. So this is not unexpected :+1: